2.9.0/final.md

```diff
@@ -0,0 +1,801 @@
+# PyTorch 2.9.0 Release Notes
+- [Highlights](#highlights)
+- [Backwards Incompatible Changes](#backwards-incompatible-changes)
+- [Deprecations](#deprecations)
+- [New Features](#new-features)
+- [Improvements](#improvements)
+- [Bug fixes](#bug-fixes)
+- [Performance](#performance)
+- [Documentation](#documentation)
+- [Developers](#developers)
+- [Security](#security)
+
+
+# Highlights
+TODO
+
+For more details about these highlighted features, you can look at the release blogpost.
+Below are the full release notes for this release.
+
+
+# Backwards Incompatible Changes
+
+## Min supported Python version is now 3.10 ([#162310](https://github.com/pytorch/pytorch/pull/162310))
+
+The minimum version of Python required for PyTorch 2.9.0 is 3.10. We also have 3.14 and 3.14t available as preview with this release.
+
+## Build metal kernels of MacOS-14+ and remove all pre-MacOS-14 specific logic, requires MacOS-14+ going forward ([\#159733](https://github.com/pytorch/pytorch/pull/159733), [\#159912](https://github.com/pytorch/pytorch/pull/159912))
+
+PyTorch MPS is only supported on MacOS-14 or later. If you need to use MPS on MacOS Ventura, please avoid updating to Python-3.9 or above
+
+## Upgrade to DLPack 1.0 ([#145000](https://github.com/pytorch/pytorch/pull/145000))
+
+This upgrade is doing the same BC-breaking changes as the DLPack release.
+Objects in `torch.utils.dlpack` have been updated to reflect these changes, such as `DLDeviceType`.
+See the PR for details on the exact changes and how to update your code.
+
+## Raise appropriate errors in `torch.cat` ([#158249](https://github.com/pytorch/pytorch/pull/158249))
+
+Raising `ValueError`, `IndexError` or `TypeError` where appropriate instead of the generic `RuntimeError`.
```

Comment:
"torch.cat now raises `ValueError`, `IndexError` or `TypeError` where appropriate instead of the generic `RuntimeError`"